src/test/java/racingcar/view/RacingCarInputExceptionTest.java

```diff
@@ -0,0 +1,69 @@
+package racingcar.view;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.ArrayList;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class RacingCarInputExceptionTest {
+
+	@Test
+	@DisplayName("null 값 테스트")
+	void isNull() {
+		String test = null;
+
+		Assertions.assertThrows(IllegalArgumentException.class, () -> RacingCarInputException.isNull(test));
+	}
```

Comment:
static import를 해주시면 더 깔끔할 것 같습니다.
```suggestion
		assertThrows(IllegalArgumentException.class, () -> RacingCarInputException.isNull(test));

```

Reply:
넵 수정하겠습니다!